packages/block-editor/src/components/iframe/index.js

```diff
@@ -239,7 +240,12 @@ function Iframe( {
 	// We need to counter the margin created by scaling the iframe. If the scale
 	// is e.g. 0.45, then the top + bottom margin is 0.55 (1 - scale). Just the
 	// top or bottom margin is 0.55 / 2 ((1 - scale) / 2).
-	const marginFromScaling = ( contentHeight * ( 1 - scale ) ) / 2;
+	// const maxHeight = props.style?.maxHeight || 0;
+	const maxContentHeight =
+		maxHeight && contentHeight > maxHeight
+			? maxHeight / ( 1 - scale )
```

Comment:
Scale the maxHeight.

Reply:
Wondering if we could get the container height using DOM API here instead 🤷🏻 

Would save all the ref passing and other chicanery